### PR TITLE
[PM-5715] Route to import page when clicking on import items button

### PR DESCRIPTION
**Pull request by djsmith85**
## Type of change
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

When tapping on the Import items button, the user gets asked if they want to get redirected to the web vault to import the items.

Clicking yes, opens the web-vault in the browser, but after logging in, they are on the vault page instead of the tools/import page.

## Code changes

* **src/App/Pages/Settings/VaultSettingsPageViewModel.cs:** Extend the existing url with "/#/tools/import" to route to the import page

## Before you submit
- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by aj-rosado — APPROVED** [review]
LGTM

File: src/App/Pages/Settings/VaultSettingsPageViewModel.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public VaultSettingsPageViewModel()
 
         private async Task GoToImportItemsAsync()
         {
-            var webVaultUrl = _environmentService.GetWebVaultUrl();
+            var webVaultUrl = _environmentService.GetWebVaultUrl() + "/#/tools/import";
```
Comment (fedemkr) [inline src/App/Pages/Settings/VaultSettingsPageViewModel.cs:43]: @djsmith85 @aj-rosado  we have a file for [External link constants](https://github.com/bitwarden/mobile/blob/main/src/Core/ExternalLinksConstants.cs#L3) so we have them in one place. I think we should change this to follow the same pattern as in other places like adding the formatted string in the external links constants file:
```c#
public const string WEB_VAULT_TOOLS_IMPORT_FORMAT = "{0}/#/tools/import";
```
and change it here as well:
```c#
var webVaultUrl = string.Format(ExternalLinksConstants.WEB_VAULT_TOOLS_IMPORT_FORMAT, _environmentService.GetWebVaultUrl());
```

Comment (djsmith85) [reply src/App/Pages/Settings/VaultSettingsPageViewModel.cs:43]: Aww cheers @fedemkr. Thanks for the info and I've created https://github.com/bitwarden/mobile/pull/2939 as a followup to address that.

Comment (fedemkr) [reply src/App/Pages/Settings/VaultSettingsPageViewModel.cs:43]: Awesome thank you! 👍

**Pull request merged**